### ISSUE TITLE
Pop from empty list

### ISSUE THREAD

**Issue by Liam-Deacon** [codejamninja/sphinx-markdown-builder#11]
Getting following exception whilst running `make markdown` on `sphinx-markdown-builder==0.4.0`:

```bash
~/.local/lib/python3.6/site-packages/sphinx_markdown_builder/markdown_writer.py", line 312, in depart_row
    self.rows.pop()
IndexError: pop from empty list
```

Currently my hack is to change line 312 to: `None if not self.rows else self.rows.pop()`

**Comment by clayrisser** [comment]
Please update to `v0.4.1`

**Comment by clayrisser** [comment]
This has been fixed with pull request #12
